tests/integration/test_dbfs_store/test_dbfs_store.py

```diff
@@ -0,0 +1,209 @@
+# Copyright 2023 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os
+import uuid
+from pathlib import Path
+from typing import List
+
+import pandas as pd
+import pytest
+import yaml
+from databricks.sdk import WorkspaceClient
+
+import mlrun
+import mlrun.errors
+from mlrun.utils import logger
+
+PARQUETS_DIR = "/parquets"
+CSV_DIR = "/csv"
+here = Path(__file__).absolute().parent
+config_file_path = here / "test-dbfs-store.yml"
+with config_file_path.open() as fp:
+    config = yaml.safe_load(fp)
+
+test_file_path = here / "test.txt"
+json_path = here / "test_data.json"
+parquet_path = here / "test_data.parquet"
+additional_parquet_path = here / "additional_data.parquet"
+csv_path = here / "test_data.csv"
+additional_csv_path = here / "additional_data.csv"
+with open(test_file_path, "r") as f:
+    test_string = f.read()
+
+MUST_HAVE_VARIABLES = ["DATABRICKS_TOKEN", "DATABRICKS_HOST"]
+
+
+def is_dbfs_configured():
+    env_params = config["env"]
+    for necessary_variable in MUST_HAVE_VARIABLES:
+        if env_params.get(necessary_variable, None) is None:
+            return False
+    return True
+
+
+@pytest.mark.skipif(
+    not is_dbfs_configured(),
+    reason="DBFS storage parameters not configured",
+)
+class TestDBFSStore:
+    def setup_class(self):
+        self._databricks_host = config["env"].get("DATABRICKS_HOST")
+        self.test_root_dir = "/test_mlrun_dbfs_objects"
+        self._object_file = f"file_{str(uuid.uuid4())}.txt"
+        self._object_path = f"{self.test_root_dir}/{self._object_file}"
+        self._dbfs_url = "dbfs://" + self._databricks_host
+        self._object_url = self._dbfs_url + self._object_path
+        self.secrets = {}
+        token = config["env"].get("DATABRICKS_TOKEN", None)
+        self.secrets["DATABRICKS_TOKEN"] = token
+        self.parquets_dir = PARQUETS_DIR
+        self.csv_dir = CSV_DIR
+        self.workspace = WorkspaceClient(host=self._databricks_host, token=token)
```

Comment:
Pretty sure this can be simplified
```suggestion
        self.workspace = WorkspaceClient()
```